src/js/app/actions.js

```diff
@@ -207,6 +207,7 @@ export const actions = (dispatch, query, state) => ({
         index = limit(index, 0, state.items.length - 1);
         if (currentIndex === index) return;
         state.items.splice(index, 0, state.items.splice(currentIndex, 1)[0]);
+        dispatch('DID_REORDER_ITEMS', { items: getActiveItems(state.items) });
```

Comment:
This will also fire if the order is changed using the FilePond API, not sure if that is an issue perse, if it is, we need to move this to the drag item method.